docs/installation/SUSE.md

```diff
@@ -8,31 +8,48 @@ parent = "smn_linux"
 +++
 <![end-metadata]-->
 
-# openSUSE
+# openSUSE and SUSE Linux Enterprise
 
-Docker is available in **openSUSE 12.3 and later**. Please note that due
-to its current limitations Docker is able to run only **64 bit** architecture.
+This page provides instructions for installing and configuring Docker on openSUSE
+and SUSE systems.
 
-Docker is not part of the official repositories of openSUSE 12.3 and
-openSUSE 13.1. Hence  it is necessary to add the [Virtualization
-repository](https://build.opensuse.org/project/show/Virtualization) from
-[OBS](https://build.opensuse.org/) to install the `docker` package.
+## Prerequisites
 
-Execute one of the following commands to add the Virtualization repository:
+Due to its current limitations Docker is able to run only **64 bit** architecture.
```

Comment:
@flavio Thank you for your contribution.   Characterizing this as a "limitation" is a bit to editorial. Simply state without the bold:

You must be running on a 64 bit architecture to install Docker on SUSE.